client/vnctl/lib/vnctl/cli/base.rb

```diff
@@ -242,7 +242,7 @@ def self.define_mode_relation(mode_type, required_opts = [], &block)
           }
 
           yield self if block_given?
-          desc "del #{base_uuid.upcase}_UUID" , "Removes a(n) #{mode_type} #{base_uuid}."
+          desc "del #{base_uuid.upcase}_UUID --option=OPTION", "Removes a(n) #{mode_type} #{base_uuid}."
```

Comment:
If anything it should have been '--option foo'.

More of an issue is that we don't really make it clear in the help text how to pass additional options, and '--foo=bar' is much more common in cli than '--foo bar'.

Should probably not be in this pull request.